bin/openshift-operations

```diff
@@ -12,5 +12,10 @@ TopologicalInventoryApiClient.configure do |config|
   config.host = "#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"]}"
 end
 
+SourcesApiClient.configure do |config|
+  config.scheme = "http"
```

Comment:
This will also be provided as an ENV var, although it's unlikely to change

Reply:
Updated to use `ENV["SOURCES_SCHEME"]`